x-pack/plugins/elastic_assistant/server/lib/langchain/execute_custom_llm_chain/index.test.ts

```diff
@@ -145,24 +152,4 @@ describe('callAgentExecutor', () => {
       status: 'ok',
     });
   });
-
-  it('throws an error if ELSER model is not installed', async () => {
```

Comment:
note: replacement test to come in a follow-up since control flow logic has changed here.

Reply:
the following replacement test:

```ts
  it('returns null if modelExists is false (the ELSER model is not installed)', () => {
    const tool = getEsqlLanguageKnowledgeBaseTool({
      assistantLangChain: true,
      chain,
      modelExists: false, // <-- ELSER model is not installed
    });

    expect(tool).toBeNull();
  });
```

is now added to `x-pack/plugins/elastic_assistant/server/lib/langchain/tools/esql_language_knowledge_base/get_esql_language_knowledge_base_tool.test.ts`.

The replacement test asserts the new expected behavior:
- when the ELSER model is not installed (`modelExists` is `false`) the `getEsqlLanguageKnowledgeBaseTool` returns `null`
- an error will NOT thrown when the ELSER model is not installed